setup.cfg

```diff
@@ -60,6 +60,7 @@ install_requires =
     certifi>=2017.4.17
     typing_extensions>=4.3,<5
     filelock>=3.5,<4
+    sortedcontainers>=2.4.0
```

Comment:
have you reached out to Security/Legal team for 3rd party lib usage?

Reply:
I did. Should've updated it here as well: https://snowflakecomputing.atlassian.net/browse/CASEC-8463